MatrixSDK/Utils/Media/MXMediaLoader.m

```diff
@@ -165,11 +156,7 @@ - (void)downloadMediaFromURL:(NSString *)url
         [request setValue:value forHTTPHeaderField:key];
     }];
     
-    // Modified by Nedap. Set the authorization token on requests (BER-229)
```

Comment:
Check this modification

Reply:
Yes it's correct, because Matrix now adds `[request setValue: [NSString stringWithFormat:@"Bearer %@", _accessToken] forHTTPHeaderField: @"Authorization"];` (the same thing I was doing here), so I kept Matrix changes.